parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -55,7 +55,8 @@ def __init__(self,
                  max_workers=float('inf'),
                  uid=None,
                  heartbeat_threshold=120,
-                 heartbeat_period=30):
+                 heartbeat_period=30,
+                 poll_period=1):
```

Comment:
Is it intentionally set to 1 that is different from 10 in the manager?

Reply:
No. I've fixed this.